packages/core/src/contents/client/getSearchContents.js

```diff
@@ -11,10 +23,15 @@ import join from 'proper-url-join';
  * @param {string} query.spaceCode - The space the content belongs to (website|mobileapp|emailTool...).
  * @param {string} query.environmentCode - The environment identifier (live | preview).
  * @param {string} query.contentTypeCode - The content type unique code (page|post|menu|pages|posts|widgets|waterproof...).
- * @param {string | string[]} query.codes - List of codes that representing the content code (about-us|today-news|header|productId...).
- * @param {number} query.page - Number of the page to get, starting at 1. The default is 1.
- * @param {number} query.pageSize - Size of each page, as a number between 1 and 180. The default is 60.
+ * @param {string | string[]} [query.codes] - List of codes that representing the content code (about-us|today-news|header|productId...).
```

Comment:
Remove spaces on param types to be consistent with other definitions 

```suggestion
 * @param {string|string[]} [query.codes] - List of codes that representing the content code (about-us|today-news|header|productId...).
```

Reply:
Done.